### PR TITLE
[P2022-1194] 3 dots menu corrections

### PR DESCRIPTION
**Pull request by jprymak**
[https://tracker.intive.com/jira/browse/P2022-1194](https://tracker.intive.com/jira/browse/P2022-1194)

- inserted label text into Typography component and changed color to #626262
- removed leftover white spaces on hover

SWEET BONUS:
-tertiary custom shadow ;)

### CODE REVIEW

File: src/components/ThreeDotsMenu/ThreeDotsMenu.style.tsx [hunk 1]
```diff
@@ -4,23 +4,21 @@ import IconButton from "@mui/material/IconButton";
...
 export const StyledMenu = styled(Menu)(({ theme }) => ({
   "& .MuiPaper-root": {
-    padding: "10px 0px",
-    color: theme.palette.grey["600"],
     borderRadius: 4,
-    boxShadow: "0px 1px 2px rgb(98 98 98 / 0.24)",
+    boxShadow: theme.customShadows.tertiary,
   },
   "& .MuiSvgIcon-root": {
     color: theme.palette.grey["500"],
```
Comment (KarolCiesluk) [inline src/components/ThreeDotsMenu/ThreeDotsMenu.style.tsx:11]: Isn't the icon supposed to have the same color as text? And probably you don't need to pass the value as string. So this should be enough: `grey[600]`

**Review by KarolCiesluk — APPROVED** [review]
Nice! I think you can merge once you fix the icons color 👍

File: src/components/ThreeDotsMenu/ThreeDotsMenu.style.tsx [hunk 2]
```diff
@@ -4,23 +4,27 @@ import IconButton from "@mui/material/IconButton";
...
   },
+  "& .MuiTypography-root": {
+    color: theme.palette.grey["600"],
+  },
   "& .MuiList-root": {
     padding: 0,
   },
   "& .MuiButtonBase-root": {
-    padding: "2px 8px",
+    padding: "6px 8px 6px 8px",
```
[the remark this answers is not in the document]
Comment (jprymak) [reply src/components/ThreeDotsMenu/ThreeDotsMenu.style.tsx:21]: Sure, thanks, but i don't think it's worth another commit just to change that one line ;)

File: src/components/ThreeDotsMenu/ThreeDotsMenu.style.tsx (hunk 1 above)
Comment (jprymak) [reply src/components/ThreeDotsMenu/ThreeDotsMenu.style.tsx:11]: Actually, icon is gray 6 and text is gray 7 according to figma project :) Thanks for this dynamic property key tip, good to know

**Pull request merged**